### PR TITLE
Store File Info

### PR DESCRIPTION
**Pull request by noahalma**
**Please describe the changes this PR makes and why it should be merged:**
nice mod cool 
**Status**

- [x] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: actions/store_file_info_MOD.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      <option value='File Content'>File Content</option>
+      <option value='File Name'>File Name</option>
+      </select>
+      <br>
+      Store in:<br>
+      <select class='round' id='storage'>
+      ${data.variables[0]}
+      </select><br>
+      Variable name:<br>
+      <input class='round' id='varName' />`
```
Comment (almeidx) [inline actions/store_file_info_MOD.js:59]:
```suggestion
File path (example: <strong>./bot.js</strong>):
<input class='round' id='filePath' /><br>
Info:
<select class='round' id='info'>
  <option value='File Size'>File Size</option>
  <option value='File Extension'>File Extension</option>
  <option value='File Character Count'>File Character Count</option>
  <option value='File Creation Date Timestamp'>File Creation Date Timestamp</option>
  <option value='File Exists'>File Exists</option>
  <option value='File Content'>File Content</option>
  <option value='File Name'>File Name</option>
</select><br>
Store in:<br>
<select class='round' id='storage'>
  ${data.variables[0]}
</select><br>
Variable name:<br>
<input class='round' id='varName' />`
```

File: actions/store_file_info_MOD.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
+module.exports = {
+  name: 'Store File Info',
+  section: 'File Stuff',
+  subtitle (data) {
```
Comment (almeidx) [inline actions/store_file_info_MOD.js:4]:
```suggestion

  subtitle (data) {
```

File: actions/store_file_info_MOD.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        break
+      case 'File Content':
+        dataType = 'String'
+        break
+      case 'File Name':
+        dataType = 'String'
+        break
+    }
+    return ([data.varName, dataType])
+  },
```
Comment (almeidx) [inline actions/store_file_info_MOD.js:37]:
```suggestion
  },
  
```

File: actions/store_file_info_MOD.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+      </select>
+      <br>
+      Store in:<br>
+      <select class='round' id='storage'>
+      ${data.variables[0]}
+      </select><br>
+      Variable name:<br>
+      <input class='round' id='varName' />`
+  },
+  init () {},
```
Comment (almeidx) [inline actions/store_file_info_MOD.js:61]:
```suggestion

  init () {},

```

File: actions/store_file_info_MOD.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+  action (cache) {
+    const data = cache.actions[cache.index]
+    const storage = parseInt(data.storage)
+    const info = data.info
+    const path = require('path')
+    const fs = require('fs')
+    const varName = this.evalMessage(data.varName, cache)
+    const filePath = this.evalMessage(data.filePath, cache)
+    if (!filePath) {
+      console.log('Insert a file path!')
```
Comment (almeidx) [inline actions/store_file_info_MOD.js:71]: Use `this.displayError()` instead

File: actions/store_file_info_MOD.js [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+        case 'File Name':
+          result = path.basename(filePath)
+          break
+      }
+      this.storeValue(result, storage, varName, cache)
+    }
+    this.callNextAction(cache)
+  },
+  mod () {
+  }
```
Comment (almeidx) [inline actions/store_file_info_MOD.js:102]:
```suggestion

  mod () {}
```

File: actions/store_file_info_MOD.js [hunk 7]
```diff
@@ -0,0 +1,103 @@
+module.exports = {
+  name: 'Store File Info',
+  section: 'File Stuff',
+  subtitle (data) {
+    return `Store File Info - ${data.info}`
+  },
+  fields: ['filePath', 'info', 'storage', 'varName'],
```
Comment (almeidx) [inline actions/store_file_info_MOD.js:7]:
```suggestion

  fields: ['filePath', 'info', 'storage', 'varName'],

```

File: actions/store_file_info_MOD.js [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+  variableStorage (data, varType) {
+    const type = data.storage
+    if (type !== varType) return
```
Comment (almeidx) [inline actions/store_file_info_MOD.js:10]:
```suggestion
    if (data.storage !== varType) return
```

File: actions/store_file_info_MOD.js [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+    const info = data.info
+    let dataType = 'Unknown type'
+    switch (info) {
```
Comment (almeidx) [inline actions/store_file_info_MOD.js:13]:
```suggestion
    let dataType = 'Unknown type'
    switch (data.info) {
```

**Comment by noahalma** [comment]
There are no trailing spaces

**Pull request merged**